ui/plugins/controller.js

```diff
@@ -87,7 +87,7 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta = $scope.job.get_current_eta();
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
-                    $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
+                    $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision;
```

Comment:
This is a good change but does not belong in this PR. Could you open up a new PR for this? I don't think we need to file a bug, since it's just a trivial change.